files/class-vip-filesystem-stream-wrapper.php

```diff
@@ -523,8 +552,10 @@ protected function string_to_resource( $data ) {
 		// Create a temporary file
 		$tmp_handler = tmpfile();
 		if ( false === fwrite( $tmp_handler, $data ) ) {
-			trigger_error( "Error creating temporary resource for data:\n" . $data,
-				E_USER_ERROR );
+			trigger_error(
+				sprintf( "Error creating temporary resource for data:\n%s\n#vip-go-streams", $data ),
```

Comment:
What's the reason for the newlines? 

```suggestion
				sprintf( "Error creating temporary resource for data: %s #vip-go-streams", $data ),
```

Might be better to just remove them for consistency?

Reply:
I put them in there because that error will be called with the data that we're trying to write to file. That data can be pretty arbitrary so the newlines will make the error logs easier to read.

Now that I think about it though, depending on the type file, that `data` variable could pretty much result in a gigantic wall of nonsense in the logs. Not sure if we should even include them in the logs now.

Reply:
Yeah, probably better to remove it.